src/test/fuzz/orphanage.cpp

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2020-2021 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <random.h>
+#include <primitives/transaction.h>
+#include <test/util/script.h>
+#include <test/fuzz/FuzzedDataProvider.h>
+#include <test/fuzz/fuzz.h>
+#include <test/fuzz/util.h>
+#include <txorphanage.h>
+
+
+FUZZ_TARGET(orphanage)
+{
+    FuzzedDataProvider fuzzed_data_provider(buffer.data(), buffer.size());
+
+    TxOrphanage orphanage;
+    std::set<uint256> orphan_work_set;
+
+    LIMITED_WHILE(fuzzed_data_provider.ConsumeBool(), COINBASE_MATURITY)
```

Comment:
any reason to use `COINBASE_MATURITY` here? Shouldn't this be a constant derived from `TxOrphanage` internals? For example a bit larger than the maximum set size?

Reply:
Already changed to `DEFAULT_MAX_ORPHAN_TRANSACTIONS`